svgpathtools/path.py

```diff
@@ -6,6 +6,8 @@
 from __future__ import division, absolute_import, print_function
 from math import sqrt, cos, sin, acos, asin, degrees, radians, log, pi, ceil
 from cmath import exp, sqrt as csqrt, phase
+from scipy.optimize import newton
```

Comment:
Why does this need `newton` from a new and extra, hard dependency? The code in the `bbox()` function already does this. Find the extreme point in both x and y to figure the bounding box. 
````python
n = self.start[0] - self.control[0]
d = self.start[0] - 2 * self.control[0] + self.end[0]
if d != 0:
     t = n / d
else:
    t = 0.5
````

Then split(t)

Reply:
No problem this can be changed. I am unfamiliar with pull request how should I do it?

Reply:
As i am completely not expert in this package I might have not seen that the bbox function could be used.

Reply:
However the code you propose above does not apply to arc

Reply:
for the arc, the function point_to_t does not work for arcs with rotation != 0. so I do not see how it is possible to go from the extremum - coordinates to the t to be able to split;  but again I am not an expert/